distribution/integ-test-zip/build.gradle

```diff
@@ -61,4 +62,11 @@ publishing {
   }
 }
 
+integTestRunner {
+  if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+    // override the suite timeout to 30 mins for windows, because it has the most inefficient filesystem known to man
+    systemProperty 'tests.timeoutSuite', '1800000!'
```

Comment:
I think that we should only override the suite timeout if it's not already overridden on the command line.